pkg/cmd/server/certs/mint_signercert.go

```diff
@@ -0,0 +1,62 @@
+package certs
+
+import (
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/origin/pkg/cmd/server/crypto"
+)
+
+type MintSignerCertOptions struct {
+	CertFile   string
+	KeyFile    string
+	SerialFile string
+	Name       string
+
+	Overwrite bool
+}
+
+func BindSignerCertOptions(options *MintSignerCertOptions, flags *pflag.FlagSet, prefix string) {
+	flags.StringVar(&options.CertFile, prefix+"cert", "openshift.local.certificates/ca/cert.crt", "The certificate file.")
+	flags.StringVar(&options.KeyFile, prefix+"key", "openshift.local.certificates/ca/key.key", "The key file.")
+	flags.StringVar(&options.SerialFile, prefix+"serial", "openshift.local.certificates/ca/serial.txt", "The serial file that keeps track of how many certs have been signed.")
+	flags.StringVar(&options.Name, prefix+"name", "openshift-signer", "The name of the signer.")
```

Comment:
default this to something that incorporates a timestamp... if the same client (e.g. a browser) gets two different certs from the same CA name with the same index, they will choke (rightfully... that's supposed to be impossible)